src/reference-assemblies/build.proj

```diff
@@ -1,15 +1,15 @@
 <Project>
+  <Import Project="../../dir.props" />
   <ItemGroup>
     <AllProjects Include="**/*.csproj" />
     <AllProjects Remove="nuspecs/pack.csproj" />
   </ItemGroup>
 
   <Target Name="Restore" />
   <Target Name="Build">
+    <Copy SourceFiles="$(MSBuildProjectDirectory)/_._" DestinationFolder="$(IntermediatePath)reference-assemblies/" />
     <MSBuild Projects="@(AllProjects)" Targets="Build" BuildInParallel="true" />
   </Target>
 
-  <Target Name="Pack">
-    <MSBuild Projects="nuspecs/pack.csproj" Targets="Pack" Properties="NuspecFile=%(AllNuSpecs.Identity)" />
-  </Target>
+  <Target Name="Pack"/>
```

Comment:
?? Do we need this target anymore?  Can it just be deleted?

Reply:
No. I removed it.